src/components/accession2/view/Accession2View.tsx

```diff
@@ -431,9 +431,13 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
               >
                 {accession.state}
               </Typography>
-              <IconButton sx={{ marginLeft: 3, height: '24px' }} onClick={() => setOpenEditStateModal(true)}>
-                <Icon name='iconEdit' className={`${classes.editIcon} edit-icon`} />
-              </IconButton>
+              {accession.state !== 'Awaiting Check-In' ? (
+                <IconButton sx={{ marginLeft: 3, height: '24px' }} onClick={() => setOpenEditStateModal(true)}>
+                  <Icon name='iconEdit' className={`${classes.editIcon} edit-icon`} />
+                </IconButton>
+              ) : (
+                <Box sx={{ marginLeft: 3, height: '24px', width: 2 }} />
```

Comment:
i could remove this box and just add the spacing to the parent Box conditionally if needed.

Reply:
This if fine, I think it's more self-explanatory